### PR TITLE
Feat/database setup

### PR DESCRIPTION
**Pull request by tiffaphm**
Basic database schema set up

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+
+CREATE TABLE user_hits (
+  id            SERIAL UNIQUE NOT NULL PRIMARY KEY,
+  user_id       INT NOT NULL,
+  visit_id      SERIAL UNIQUE NOT NULL,
```
Comment (tylertruong) [inline database/schema.sql]: Will these end up being foreign keys? If so you will need to specify that.

File: database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  action_id     SERIAL UNIQUE NOT NULL,
+  search_id     INT,
+  booking_id    SERIAL UNIQUE,
+  hit_time      TIME NOT NULL,
+  hit_date      DATE NOT NULL
```
Comment (tylertruong) [inline database/schema.sql]: You can store the date object in ISO format to avoid this: 2017-10-01T23:59:59Z

File: database/index.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+connect.db()
```
Comment (tylertruong) [inline database/index.js]: This connect.db() does nothing.

File: database/schema.sql (hunk 2 above)
Comment (tiffaphm) [reply database/schema.sql]: I'm pre-formatting the time and date before inserting into the DB.

**Pull request merged**